components/Budget/ExpenseSummary.jsx

```diff
@@ -0,0 +1,88 @@
+                        />
+                        <View className="ml-2 flex-1">
+                            <Subtitle
+                                extraClassName={expense.placeholder && "text-gray-600"}
+                                oneLine
+                            >
+                                {expense.reference}
+                            </Subtitle>
+                        </View>
+                    </View>
+                    <View className="flex-row items-center ml-2">
+                        <Subtitle extraClassName={expense.placeholder && "text-gray-600"}>
+                            £{expense.cost}
+                        </Subtitle>
+                        <View className="ml-2 w-6 justify-center items-center">
+                            <Icon
+                                name={isDropdownOpen ? "caret-down" : "caret-right"}
+                                size={30}
+                                color={expense.placeholder ? "#757575" : "black"}
+                            />
+                        </View>
+                    </View>
+                </View>
+                {isDropdownOpen && (
+                    <>
+                        {isUser && (
+                            <TouchableOpacity
+                                className={`w-full h-10 rounded-md border-2 mt-4 justify-center items-center flex-row bg-slate-100`}
+                                onPress={() => {
+                                    alert("Implement editing expense");
+                                }}
+                            >
+                                <Subtitle extraClassName="mr-4">Edit Expense</Subtitle>
+                                <Icon name="pencil" size={30} color="black" />
+                            </TouchableOpacity>
+                        )}
+                        <View className="justify-between mt-2 flex-row">
+                            <View className="flex-1">
+                                <Text className="font-semibold text-base">Reference:</Text>
+                                <Text>{expense.reference}</Text>
+                                <Text className="font-semibold text-base mt-2">Description:</Text>
+                                <Text>{expense.description}</Text>
+                                <Text className="font-semibold text-base mt-2">Submitted:</Text>
+                                <Text>{timeString}</Text>
+                            </View>
+                            {expense.receipt && (
+                                <View className="flex-1 justify-center items-center">
+                                    <Text className="font-semibold text-base text-center mb-2">
+                                        Receipt:
+                                    </Text>
+                                    <View className="w-40 h-40">
+                                        <ImageViewer URL={expense.receipt} centered />
```

Comment:
think the spacing around the reciept looks slightly odd- too much at the side/ not enough at the bottom